chatlab/registry.py

```diff
@@ -161,9 +161,9 @@ def generate_function_schema(
         model = create_model(
             function.__name__,
             __config__=FunctionSchemaConfig,
-            **fields,
+            **fields,  # type: ignore
         )
-        parameters: dict = model.schema()
+        parameters: dict = model.schema()  # type: ignore
```

Comment:
Why does this type not work?

Reply:
It doesn't like the redefinition of `parameters`